src/components/application_manager/rpc_plugins/vehicle_info_plugin/src/commands/mobile/subscribe_vehicle_data_request.cc

```diff
@@ -201,20 +201,39 @@ bool SubscribeVehicleDataRequest::CheckSubscriptionStatus(
 }
 
 bool SubscribeVehicleDataRequest::SubscribePendingVehicleData(
-    ApplicationSharedPtr app, const smart_objects::SmartObject& msg_params) {
+    ApplicationSharedPtr app, smart_objects::SmartObject& msg_params) {
   LOG4CXX_DEBUG(logger_, "Subscribing to all pending VehicleData");
 
-  for (const auto& vi_name : vi_waiting_for_subscribe_) {
+  std::set<hmi_apis::Common_VehicleDataResultCode::eType> skiped_result_codes(
```

Comment:
Maybe it is good place for `using hmi_apis::Common_VehicleDataResultCode = VD_ResultCode`

Reply:
Done in commit 664d9086492f57c7307e13597b7979113f0a4ada